adapters/network.js

```diff
@@ -40,6 +42,24 @@ const networkClientConfigs = {
   httpsAgent: new https.Agent({ keepAlive: true })
 };
 
+/**
+ * check if proxy is present or not and return the options accordingly
+ * @param {*} url
+ * @param {*} options
+ * @returns
+ */
+const getProxyOptions = (url, options = {}) => {
+  if (skipProxy(url)) {
```

Comment:
When we want to skip it will return  `true` then it will add the proxyAgent which is not the expected case

If we don't want to skip then it would return false and we would want to add the agent in options